code/modules/events/disease_outbreak.dm

```diff
@@ -3,14 +3,15 @@
 	typepath = /datum/round_event/disease_outbreak
 	max_occurrences = 1
 	min_players = 10
-	weight = 5
+	weight = 10

```

Comment:
I don't think this should be increased to 10. Keep it at 5. You didn't justify it in your PR text either. Damn viro main

Reply:
actually nerfed the virology potential of this by making it spawn special diseases instead of advanced diseases. but sure.